src/tests/class_tests/openms/source/LogStream_test.cpp

```diff
@@ -87,11 +87,12 @@ START_SECTION(([EXTRA] OpenMP - test))
   Log_info.insert(stream_by_logger);
   Log_info.remove(cout);
 
-#ifdef _OPENMP
-omp_set_num_threads(8);
-#pragma omp parallel
-#endif
+
   {
+    #ifdef _OPENMP
+	omp_set_num_threads(8);
+    #pragma omp parallel for
+    #endif
     for (int i=0;i<10000;++i)
```

Comment:
what about making the `i` an unsigned int? I think otherwise its incompatible with OpenMP 2.x

Reply:
No it needs to be signed for compatibility afaik

Reply:
https://stackoverflow.com/questions/2820621/why-arent-unsigned-openmp-index-variables-allowed